### ISSUE TITLE
Spike: OpenShift monitoring to notify us when batches fail.

### ISSUE THREAD

**Issue by ianliuwk1019** [bcgov/nr-fom#698]
**Describe the task**
There was an incident recently that the FOM batch jobs failed and we were not aware until production and reported by business.
It is better to have a monitoring on OpenShift to notify us when batches failed.

Discuss what possible monitoring tools or implementation can we have to improve our awareness of the batch failure.

**Acceptance Criteria**
- [ ] [TO BE REFINED]

**Additional context**
- Related to #217.
- Seems Craig does get notification email but buried in junk emails.

**Comment by ianliuwk1019** [comment]
You are right @basilv , the very early ticket #217 is probably the one you mentioned.

**Comment by DerekRoberts** [comment]
@ianliuwk1019 I have a notifier for this that assigns issue to people in CODEOWNERS when jobs fail.  Let's hook that up and see if it drives you nuts?

**Comment by DerekRoberts** [comment]
This is now working in FOM for deployment.  If you have other uses then please request or reuse as appropriate.
https://github.com/bcgov/nr-fom/blob/main/.github/workflows/.deploy.yml#L170
